js/mail.js

```diff
@@ -1120,6 +1123,12 @@ var Mail = {
 				Mail.BackGround.messageFetcher.restart();
 				// hide message detail view on mobile
 				$('#mail-message').addClass('hidden-mobile');
+			},
+
+			onWindowResize: function() {
+				// Resize iframe
+				var iframe = $('iframe');
```

Comment:
I recommend making this more specific, to avoid accidentally catching other iframes: `$('#mail-content iframe')`